### ISSUE TITLE
Integration tests do not match the documentation

### ISSUE THREAD

**Issue by ghostd** [lingui/js-lingui#202]
Hi,
It seems the [tests](https://github.com/lingui/js-lingui/blob/master/packages/babel-plugin-extract-messages/test/fixtures/js/all.js#L7) of babel-plugin-extract-messages don't match the [documentation](babel-plugin-extract-messages).
It seems:
```js
i18n._("{count, plural, one {# book} other {# books}}", {
  values: {
    count: count
  }
})
```
should be
```js
i18n._("{count, plural, one {# book} other {# books}}", {
  count: count
})
```

**Comment by snegostup** [comment]
It's not only tests. `babel-plugin-extract-messages` itself doesn't match its documentation :)
See [this](https://github.com/lingui/js-lingui/issues/181) and [this](https://github.com/lingui/js-lingui/pull/185).

**Comment by tricoder42** [comment]
@ghostd Absolutely! Could you please send the link to documentation again? The one you've posted is invalid. Thanks!

**Comment by ghostd** [comment]
Sorry, that was a link to the migration doc:
https://github.com/lingui/js-lingui/blob/master/docs/releases/migration-2.rst#signature-of-i18n_-changed

**Comment by tricoder42** [comment]
Ah, I see, thanks!

Fixed and released in **2.0.7**. 🎉
